packages/vehicle-lifecycle-network/lib/vda.js

```diff
@@ -82,7 +82,7 @@ function scrapVehicle(scrapVehicle) {
  * @transaction
  */
 function scrapAllVehiclesByColour(scrapAllVehicles) {
```

Comment:
This should be an explicit colour (one would assume from an Enum), and not a vehicle from which you extract the colour?